mountpoint-s3-fs/src/prefetch/backpressure_controller.rs

```diff
@@ -327,10 +327,11 @@ mod tests {
     use std::sync::Arc;
 
     use futures::executor::block_on;
-    use mountpoint_s3_client::mock_client::{MockClient, MockClientError};
+    use mountpoint_s3_client::mock_client::MockClientError;
     use test_case::test_case;
 
     use crate::mem_limiter::MemoryLimiter;
+    use crate::memory::PagedPool;
 
     #[test_case(1024 * 1024 + 128 * 1024, 2)] // real config
```

Comment:
np: use `INITIAL_READ_WINDOW_SIZE` here

Reply:
(in both cases) you are right, we should use the constant. It's an unrelated change though. Would you mind opening another PR?